modin/core/dataframe/base/interchange/dataframe_protocol/utils.py

```diff
@@ -59,15 +58,14 @@ class DTypeKind(enum.IntEnum):  # noqa PR01
 class ColumnNullType(enum.IntEnum):  # noqa PR01
     """
     Integer enum for null type representation.
-
     Attributes
     ----------
     NON_NULLABLE : int
         Non-nullable column.
     USE_NAN : int
-        NaN/NaT value.
+        Use explicit float NaN value.
     USE_SENTINEL : int
-        Sentinel value besides NaN/NaT.
+        Sentinel value besides NaN.
```

Comment:
```suggestion
        Sentinel value excluding NaN.
```
besides looks like NaN is possible too, don't it?

Reply:
this is a verbatim copy of the spec from the [upstream](https://github.com/data-apis/dataframe-api/blob/main/protocol/dataframe_protocol.py), so I would redirect you to [it](https://github.com/data-apis/dataframe-api/) instead

Reply:
ok, let's leave as is for now